code/src/index.css

```diff
@@ -1,13 +1,135 @@
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: #181818;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+h1 {
+  color: #ffffff;
+  border-bottom: 1px solid #34363A;
+  padding-bottom: 10px;
 }
+
+h3 {
+  font-size: 14px;
+  color: #ffffff;
+}
+
+.albums-container {
+  width: 90%;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+}
+
+.album-card {
+  margin: 20px;
+}
+
+.album-cover-img {
+  width: 100%;
+}
+
+.album-cover {
+  position: relative;
+}
+
+.icons {
+  position: absolute;
+  z-index: 5;
+  width: 80%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%,-50%);
+  display: none;
+}
+
+.dots-button, .heart-button {
+  width: 30px;
+}
+
+.play-button {
+  width: 70px;
+}
+
+/*Hover effects*/
+/* makes Play button bigger on hover*/
+.play-button:hover {
+  width: 80px;
+  height: 80px;
+  display: inline-block;
+  margin: 10px;
+}
+
+/* puts filter on album cover on hover*/
+.album-cover-img:hover {
+  filter: brightness(0.20);
+}
+
+/* show icons on hover*/
+.album-cover:hover .icons {
+  display: flex;
+}
+
+/*Special class to keep the filter effect when hovering on the icons*/
+.icons:hover~.album-img-hover { 
```

Comment:
Not sure if you wanted to use CSS selector `~` - if yes, then we need to add whitespaces, without them your code can break (and I think it do) :

```
.icons:hover ~ .album-img-hover { ... }
```